### ISSUE TITLE
Forge 1.20.1 support?

### ISSUE THREAD

**Issue by K1ller0561** [IsaiahMC/multiworld#63]
Any chance for support Forge 1.20.1?

**Comment by xhatz** [comment]
I would absolutely appreciate that as well :(

**Comment by sandwitcherz** [comment]
Hello, same here, a Forge 1.20.1 could be a must ^^ thanks

**Comment by Bananamilk452** [comment]
You can find out 1.20.1 jar file in this repo on `output` folder - https://github.com/IsaiahMC/multiworld/blob/master/output/Multiworld-Forge-1.20.1.jar

Press view raw or Raw to download!

**Comment by Xplodin** [comment]
> You can find out 1.20.1 jar file in this repo on `output` folder - https://github.com/IsaiahMC/multiworld/blob/master/output/Multiworld-Forge-1.20.1.jar
> 
> Press view raw or Raw to download!

can this be uploaded to curse?
thank you for the link otherwise!

**Comment by Aiwen094** [comment]
There is a conflict with the Sinytra Connector mod and its prerequisite Forgified Fabric API mod, preventing it from starting.

**Comment by sashistillfrvr** [comment]
True, this doesn't work with Sinytra Connector mod. Would be cool if you add support, the mod is incredible.

**Comment by JamZia** [comment]
Any chance we're seeing support for Sinytra mod anytime soon?
